v2/src/css/custom.css

```diff
@@ -519,3 +519,45 @@ html[data-theme="dark"] .markdown blockquote > *:last-child {
     bottom: 144px !important;
   }
 }
+
+/* Styles for collapsible elements */
+details {
```

Comment:
Lets change the `change_me/collapsible-element` page to add a default detail component as well, the current ones force font size to 20 px so hard to review the output

```mdx
<details>
<summary>Default State</summary>
This is the default state of the `detail` component which uses font size 16px
</details>
```

Reply:
its updated